src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md

```diff
@@ -89,9 +90,11 @@ resource.
 
 Although many browsers report mixed content warnings to the user, by the time
 this happens, it is too late: the insecure requests have already been performed
-and the security of the page is compromised. This scenario is, unfortunately,
-quite common on the web, which is why browsers can't just block all mixed
-requests without restricting the functionality of many sites.
+and the security of the page is compromised. 
+
+This scenario has traditionally been
+quite common on the web, which is why many browsers haven't just blocked all mixed
+requests, so as not to restrict the functionality of many sites. However, as HTTPS has become [overwhelmingly adopted on the web](https://transparencyreport.google.com/https/overview?hl=en), some browsers have announced that they will begin blocking mixed content ([more below](#blocking-optionally-blockable-content)).
```

Comment:
This is my first PR, and I don't have the site staged locally, so I'm guessing at how anchor tags are autogenerated based on header text, based on experience with past Jekyll-based sites.